__tests__/app.test.js

```diff
@@ -60,3 +60,50 @@ describe('GET /api/articles/:article_id', () => {
     });
   });
 });
+
+describe('PATCH /api/articles/:article_id', () => {
+  test('Status 201: Request body accepts an object in the form `{ inc_votes: newVote }`, updates the DB and then responds with the updated article', () => {
+    const updatedArticle = {
+      article_id: 1,
+      title: expect.any(String),
+      topic: expect.any(String),
+      author: expect.any(String),
+      body: expect.any(String),
+      created_at: expect.any(String),
+      votes: 223,
+    };
+    const requestBody = {inc_votes: 123};
+
+    return request(app).patch('/api/articles/1').send(requestBody).expect(201).then(({body})=> {
```

Comment:
status 201 is for the succesful creation of something, think about if this is creating a new item or not